### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jskinne3**
## What changes

Accumulated development changes into trunk, including setting 700M memory allocation in the mainfest.yml file to address the error `app requested more memory than available` during deployment from CircleCI.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
               export PATH=$HOME/bin:$PATH
               cf login -a https://api.fr.cloud.gov -u $DEV_USER -p $DEV_PASSWORD -o $ORG_NAME -s $SPACE_NAME
               echo "Login successful."
-              cf zero-downtime-push opre-ops-test -f manifest.yml
+              cf push opre-ops-test -f manifest.yml
```
Comment (amymok) [inline .circleci/config.yml:25]: [question]: Why is `zero-downtime-push` being removed?

File: manifest.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 applications:
   - name: opre-ops-test
+    memory: 700M
```
Comment (amymok) [inline manifest.yml:4]: [todo] Can you include the reasons why we need this in the PR description?

**Review by amymok — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (jskinne3) [reply .circleci/config.yml:25]: To make deployment from local as similar as possible to CircleCI, and to reduce memory usage. I didn't confirm that the change is currently needed for a successful deployment, but I thought if we are running into memory problems it's good to start trimming down unnecessary things.

File: manifest.yml (hunk 2 above)
Comment (jskinne3) [reply manifest.yml:4]: Yes, great idea, thank you

File: .circleci/config.yml (hunk 1 above)
Comment (jskinne3) [reply .circleci/config.yml:24]: Update: at your request have have confirmed that `zero-downtime-push` blocks deployment by exceeding the memory limit. So I have restored `push` in the config file, and I have also removed the [autopilot](https://github.com/contraband/autopilot) plugin, which was [deprecated years ago](https://engineering.18f.gov/continuous-deployment/).

Comment (amymok) [reply .circleci/config.yml:24]: Okay thank you for checking, it sounds like it is necessary to remove it.

**Review by amymok — APPROVED** [review]
LGTM!

**Pull request merged**